addon/components/nrg-datetime/calendar/component.js

```diff
@@ -294,7 +294,8 @@ export default Component.extend(EKMixin, EKFirstResponderOnFocusMixin, {
 
     date[operation](dateTransformation);
 
-    const userDisabled = this.isDateDisabled && this.isDateDisabled(date);
+    const precision = Object.keys(dateTransformation)[0];
```

Comment:
Although clever, this isn't going to work in all cases, because `dateTransformation` can have multiple manipulation aspects in the same object.

It would be better to utilize these 😉 

```js
  isSelectingDays: false,
  isSelectingMonths: false,
  isSelectingYears: false,
  isSelectingHours: false,
  isSelectingMinutes: false,
```